constructor/header.sh

```diff
@@ -509,26 +490,22 @@ rm -f "$PRECONDA"
 CONDA_QUIET="$BATCH" \
 "$CONDA_EXEC" constructor --prefix "$PREFIX" --extract-conda-pkgs || exit 1
 
-#The templating doesn't support nested if statements
-#if has_pre_install
+{%- if has_pre_install %}
```

Comment:
Can the nested conditionals in this file be indented?

Reply:
Check last commits.